build-in-container/README.md

```diff
@@ -41,6 +41,9 @@ ls out/RPMS/x86_64/
 #  Run the tools manually
 make build-packages SRPM_PACK_LIST="hello_world_demo" -j$(nproc)
 
+# Clean up Mariner workspace, and docker image and container. This requires root privileges
+sudo ./CBL-MarinerTutorials/mariner-docker-builder.sh --mariner_dir /path/to/CBL-Mariner -c
```

Comment:
Just curious, can the `-c` be placed anywhere, or does it have a specific position in the arguments list?

Reply:
No, it has to be specific, that's why adding this in README

Reply:
I don't see any mention in the comment itself, so one could easily not realize `-c` must be placed at the end of the argument list.

And I do wonder if the option should be handled differently so the placement is flexible for the user, or it is at a consistent spot relative to other shorthand arguments like `-i`, which comes immediately after `./CBL-MarinerTutorials/mariner-docker-builder.sh`